public/app/features/dashboard-scene/settings/GeneralSettingsEditView.tsx

```diff
@@ -96,13 +97,18 @@ export class GeneralSettingsEditView
     this._dashboard.setState({ tags: value });
   };
 
-  public onFolderChange = (newUID: string | undefined, newTitle: string | undefined) => {
+  public onFolderChange = async (newUID: string | undefined, newTitle: string | undefined) => {
```

Comment:
Great finding! I'd suggest adding some tests here to make sure we call the updateNavModel with the new folder ID. WDYT?

Reply:
adapted tests for this